src/qaqc/01-organize_raw_data.Rmd

```diff
@@ -149,29 +149,37 @@ Load field notes and define the start time as the 15 minute preceding the field
 
 ```{r}
 field_notes <- read_excel("data/sensor_field_notes.xlsx") %>%
-  mutate(start_DT = ymd_hm(paste(date, start_time_mst, tz = "MST")),
-         start_DT = floor_date(start_DT, "15 minutes"),
+  mutate(start_DT = ymd_hm(paste(date, start_time_mst, tzone = "MST"))) %>%
+  mutate(#start_DT = with_tz(start_DT, tzone = "MST"),
+         DT_round = floor_date(start_DT, "15 minutes"),
+         DT_join = as.character(DT_round),
          site = tolower(site),
-         season = year(start_DT)) %>%
-  arrange(start_DT)
+         season = year(DT_round)) %>%
+  arrange(DT_round) %>%
+  # binary version of deployed/pulled options. may need to play around with this som
+  # more
+  mutate(where = case_when(!is.na(sensor_pulled) & !is.na(sensor_deployed) ~ 0,
+                           !is.na(sensor_pulled) & is.na(sensor_deployed) ~ 1,
+                           is.na(sensor_pulled) & !is.na(sensor_deployed) ~ 0,
+                           is.na(sensor_pulled) & is.na(sensor_deployed) ~ 0))
 ```
 
 Munge field notes
 
 ```{r}
-field_notes_clean <- field_notes %>% 
-  mutate(sonde_impact = case_when(grepl('sonde', visit_comments, ignore.case = T) ~ 'y',
-                                  grepl('calib', visit_comments, ignore.case = T) ~ 'y',
-                                  sensor_pulled == 'x' ~ 'y',
-                                  sensor_deployed == 'x' ~ 'y',
-                                  grepl('yes', sensors_cleaned, ignore.case = T) ~ 'y'
-                                  ),
-          end_DT = case_when(sensor_pulled == 'x' ~ ymd_hm(paste0(season, '-12-31 23:59'), tz = 'MST'),
-                               sensor_deployed == 'x' ~ start_DT,
-                               TRUE ~ start_DT + minutes(15)),
-         start_DT = case_when(sensor_deployed == 'x' ~ ymd_hm(paste0(season, '-01-01 00:00'), tz = 'MST'),
-                                    TRUE ~ start_DT)) %>% 
-  filter(sonde_impact == 'y', !is.na(start_DT))
+# field_notes_clean <- field_notes %>% 
+#   mutate(sonde_impact = case_when(grepl('sonde', visit_comments, ignore.case = T) ~ 'y',
+#                                   grepl('calib', visit_comments, ignore.case = T) ~ 'y',
+#                                   sensor_pulled == 'x' ~ 'y',
+#                                   sensor_deployed == 'x' ~ 'y',
+#                                   grepl('yes', sensors_cleaned, ignore.case = T) ~ 'y'
+#   ),
+#   end_DT = case_when(sensor_pulled == 'x' ~ ymd_hm(paste0(season, '-12-31 23:59'), tz = 'MST'),
+#                      sensor_deployed == 'x' ~ start_DT,
+#                      TRUE ~ start_DT + minutes(15)),
+#   start_DT = case_when(sensor_deployed == 'x' ~ ymd_hm(paste0(season, '-01-01 00:00'), tz = 'MST'),
+#                        TRUE ~ start_DT)) %>% 
```

Comment:
TZ in `print(tz(field_notes_clean$start_time_mst))` is UTC, but is MST for `start_DT` and `end_DT`.